Update Motor.cpp

fbusche opened the pull request:
added nested if statement for take off and landing. need to most likely change takeOff speed variable based on testing because we don't know how much we have to run motors to make pod lift off. also will have to make sure this if statement is continuously being checked based on the data from joystick.

fbusche committed:
Update Motor.cpp

added nested if statement for take off and landing. need to most likely change takeOff speed variable based on testing because we don't know how much we have to run motors to make pod lift off. also will have to make sure this if statement is continuously being checked based on the data from joystick.

Tpanda03 committed:
Update Motor.cpp

Tpanda03 reviewed (commented):
Moved joystick control logic to i2c file, cmakes all pass

fbusche committed:
Update Motor.cpp

took the nested if that was used for what to do when button pressed and turned into a function. now this can be repeated called in i2c.
